### DISCUSSION TITLE
Welcome to gunn-web-app Discussions!

### DISCUSSION THREAD

**Discussion by SheepTester** [Orbiit/gunn-web-app#179] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * ***S y n e r g i z e***

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SheepTester** [comment]
#179 is where it's at

**Reply by SheepTester** [reply]
Will all due respect, you're wrong.

**Comment by ky28059** [comment]
<img width="617" alt="Screen Shot 2020-12-08 at 12 05 26 PM" src="https://user-images.githubusercontent.com/60120929/101535606-bfea4b00-394d-11eb-8993-49e05b05c61c.png">
When will you stop the deceit and change the logo to a transparent background?

**Reply by SheepTester** [reply]
It is transparent
![image](https://avatars0.githubusercontent.com/u/31393736?s=200&v=4)

**Reply by ky28059** [reply]
Impossible...

**Comment by anematode** [comment]
人们！我有办法。

My Chinese has deteriorated. :(

**Reply by ky28059** [reply]
太可惜啦

mine too :(

**Comment by SheepTester** [comment]
Why can't I hide a comment as too heated??
